src/phonegap/popup.js

```diff
@@ -1,7 +1,7 @@
-import { Popup } from '../html5/popup';
+import { Popup as HTML5Popup } from '../html5/popup';
 import { Device } from './device';
 
-export class PhoneGapPopup extends Popup {
+export class Popup extends HTML5Popup {
```

Comment:
There is another place in this file that has `PhoneGapPopup`. Look for the `static open(url)` method at the bottom. That needs to change too

Reply:
Yep good catch.